.github/CODEOWNERS

```diff
@@ -2,7 +2,7 @@
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
 
 # Global Owners
-*  @gravitational/codeowners-gravity-code
+*  @gravitational/gravity-team
 
-# Web UI
-/web/ @gravitational/codeowners-gravity-web
+# Edits to this file
+/.github/CODEOWNERS @knisbet
```

Comment:
I want something like @gravitational/admins here, but no gravity team members are currently part of that group.